release/s3_sync.sh

```diff
@@ -0,0 +1,36 @@
+#!/usr/bin/env bash
+# Copyright 2021 Amazon.com Inc. or its affiliates. All Rights Reserved.
```

Comment:
I think we decided that we don't actually need the year, so we should just start removing the year from our files so that it won't change on a yearly basis.

Reply:
Fair enough, we should have one PR to remove them. Please open a ticket if this is a thing.

Reply:
Yea at least for this PR and going forward, let's remove the dates. We can open a ticket for the ones that already exist in the future, though it doesn't matter too much for ones we committed already.